rfc/0054-sdk3-management-apis.md

```diff
@@ -32,6 +32,19 @@ On the bucket interface:
 [ViewIndexManager](#viewindexmanager) ViewIndexes();
 </pre></big>
 
+On the scope interface:
+
+<big><pre>
+[ScopeSearchIndexManager](#scopesearchindexmanager) SearchIndexes();
+</pre></big>
+
+
+On the collection interface:
```

Comment:
Is this bit meant to be here?

Reply:
Yeah, I should have added this in the QueryIndexManager PR.  It's not strictly related to this change, but it's hard enough getting a review of a management PR as it is, without adding a 2nd one...